tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "sourceMap": true,
+    "target": "es5",
+    "outDir": "lib",
+    "declaration": true
+  },
+  "files": [
+    "src/mediator.ts",
```

Comment:
For the implicit anys, it might not work as a regular import statement, and you'll have to do the old fashioned `const foo = require('foo');`